Makefile

```diff
@@ -312,10 +312,16 @@ $(TARBALL): release-only $(NODE_EXE) doc
 	cp -r out/doc/api/* $(TARNAME)/doc/api/
 	rm -rf $(TARNAME)/deps/v8/test # too big
 	rm -rf $(TARNAME)/doc/images # too big
+	rm -rf $(TARNAME)/deps/zlib/contrib # too big, unused
 	find $(TARNAME)/ -type l | xargs rm # annoying on windows
 	tar -cf $(TARNAME).tar $(TARNAME)
 	rm -rf $(TARNAME)
-	gzip -f -9 $(TARNAME).tar
+	gzip -k -f -9 $(TARNAME).tar
```

Comment:
`-k` isn't very portable, it's missing on older versions of gzip and we like to make some of these assets on older systems. Perhaps `gzip -c -f -9 $(TARNAME).tar > $(TARNAME).tar.gz`